src/main/webapp/search-script.js

```diff
@@ -80,5 +80,47 @@ function addCourse() {
   url.searchParams.set("num-units", units);
   url.searchParams.set("term", termName);
   url.searchParams.set("school-name", schoolName);
-  fetch(url, { method: "POST" });
+  fetch(url, {method:"POST"});
+}
+
+function onSignIn(googleUser) {
+  const profile = googleUser.getBasicProfile();
+  document.getElementById("class-info").classList.remove("hidden");
+  document.getElementById("login-box").classList.add("hidden");
+  document.getElementById("school-name").innerHTML = `Hi, ${profile.getName()}! Your email is ${profile.getEmail()}`;
+  const token = googleUser.getAuthResponse().id_token;
+  const url = new URL("/secure", window.location.origin);
+  url.searchParams.set("token", token);
+  fetch(url, {method:"POST"})
+    .then(response => response.json()) 
+    .then((id) => {
+        if (id == "Unsecure email.") {
+            signOut();
```

Comment:
Technically this isn't signing out, they were never signed in to begin with (hence why you probably don't want to hide/unhide stuff yet).